<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskManageClass</class>
 <widget class="QMainWindow" name="TaskManageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1698</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TaskManage</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="usernameInput">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>230</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordInput">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>270</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="LoginBTN">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>310</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1698</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TaskManage.qrc"/>
 </resources>
 <connections/>
</ui>
